packages/protocol/contracts/governance/Governance.sol

```diff
@@ -148,6 +148,8 @@ contract Governance is
     uint256 weight
   );
 
+  event AccountProposalsVoteRevoked(address indexed account);
```

Comment:
```suggestion
  event ProposalVoteRevoked(
    uint256 indexed proposalId,
    address indexed account,
    uint256 value,
    uint256 weight
  );
```

for consistency with `ProposalUnvoteRevoked` and `ProposalVoted`
We should be able to tell from these events the canonical vote tallies on a specific proposal and voter